buildout.cfg

```diff
@@ -2,6 +2,7 @@
 extends =
     https://raw.github.com/collective/buildout.plonetest/master/test-4.3.x.cfg
     https://raw.github.com/collective/buildout.plonetest/master/qa.cfg
+    https://raw.githubusercontent.com/plone/plone.app.robotframework/e1b2796/versions.cfg
```

Comment:
@hvelarde I think these versions are only compatible with Plone 5. See this commit:

https://github.com/plone/plone.app.robotframework/pull/71/commits/160275e49e68c234e6d4f695cadaf2dc98bae7cd

 that is in the same PR.